cmd/controller/main.go

```diff
@@ -43,6 +50,19 @@ func main() {
 	lo.Must0(op.AddHealthzCheck("cloud-provider", awsCloudProvider.LivenessProbe))
 	cloudProvider := metrics.Decorate(awsCloudProvider)
 
+	serverVersion, err := op.VersionProvider.Get(ctx)
+	if err != nil {
+		logging.FromContext(ctx).Error(err)
+	}
+
+	floatServerVersion, err := strconv.ParseFloat(serverVersion, 32)
```

Comment:
```suggestion
	floatServerVersion := lo.Must0(strconv.ParseFloat(serverVersion, 32))
```

Is it possible for this ParseFloat to fail?

Reply:
No, as we will always get the version back as a float string